src/nvim/version.c

```diff
@@ -63,11 +63,7 @@ static char *features[] = {
   "-acl",
 #endif
 
-#if defined(HAVE_ICONV)
   "+iconv",
-#else
-  "-iconv",
-#endif
 
   "+tui",
```

Comment:
this also is unnecessary. in theory, FEAT_TUI was useful for avoiding GPL v3 https://github.com/neovim/neovim/issues/9933 , but that issue is parked until we can replace unibilium with https://github.com/leonerd/libtickit